pkg/proxy/cache.go

```diff
@@ -0,0 +1,22 @@
+package proxy
+
+import (
+	"github.com/codeready-toolchain/registration-service/pkg/application"
+	"github.com/codeready-toolchain/registration-service/pkg/proxy/namespace"
+	"github.com/gin-gonic/gin"
+)
+
+type UserNamespaces struct {
+	app application.Application
+}
+
+func NewUserNamespaces(app application.Application) *UserNamespaces {
```

Comment:
this is a struct with a single method. Could we refactor this to a func instead?

Reply:
We still need a struct to implement the cache. For now it's just a place holder for the cache we start working on when done with this PR.